### DISCUSSION TITLE
Copilot unable to connect to completions API

### DISCUSSION THREAD

**Discussion by drewwalton19216801** [community/community#8441] [Copilot Conversations]
Github Copilot in VS Code is not working, and I see the following error in the developer tools console:

`POST https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions timed out after 30000 ms: Error: POST https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions timed out after 30000 ms`

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by drewwalton19216801** [reply]
> Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

No, I am not. Please let me know what logs, if any, would be helpful in diagnosing this issue.

**Reply by davecheney** [reply]
Make sure you’re using the latest extension. If you see an exclamation point over the copilot icon on the status bar hover over it for more information, click it open the debug log.

**Reply by drewwalton19216801** [reply]
Seems to be working again, possibly could have been a network issue on my end.
